client/src/components/create-assignment/ParentAssignment.js

```diff
@@ -73,6 +88,7 @@ const ParentAssignment = ({ indices, addSubAssignments, setAssignments, assignme
                   assignments={assignments} 
                   setAssignments={setAssignments} 
                   removeAssignment={removeAssignment}
+                  formulaAttributeNames={subFormulaAttributeNames ? subFormulaAttributeNames : []}
```

Comment:
I believe that subFormulaAttributeNames ?? [] should work here

Reply:
Thanks for the tip but if I use this then I get the following error: `Invalid prop `formulaAttributeNames` of type `string` supplied to `Assignment`, expected `array`.`